src/libs/common/Utils.ts

```diff
@@ -4,7 +4,7 @@ import * as _ from 'lodash';
 import request from 'request';
```

Comment:
Hey hate to be that guy but this change probably not as important to the functionality you want. I suggest just sticking to the replace functionality - if you want to have a different response in the message I'd prefer a separate PR for that after this one.

Reply:
Ah. Should I revert it back? I just thought that it would be nice to print out the playlist name, but it definitely isn't that important. Also, I suggest we may want a loop functionality in queue. I'll add a new pull request if I decide to implement that.

Reply:
Yes I think this would be better saved for a future PR :)